.github/ISSUE_TEMPLATE/question.md

```diff
@@ -6,3 +6,29 @@ labels: question
 assignees: ''
 
 ---
+
+To get quick replies, users are recommended to create clear question following the guideline below.
+
+#### Prerequisites
+
+- Make sure it's a Ghidra relevant question
+
+#### Description
+
+#### Steps to reproduce (optional)
+
+- Input data (source and sample data)
+- Software version
+- Code/commands
+- Output/Error message
+
+#### What have you tried?
+
+- Have you Googling?
+- Have you searched similar issue on this site?
+
+#### Guide to comment
+
+- DO NOT create new answer to reply any comment or reply
```

Comment:
The intention here is unclear.

Reply:
The intention is do not create new issue for reply a answer

Reply:
If I understand correctly you mean that users should comment on an existing issue related to their question instead of opening a new issue?

Reply:
Yes

Reply:
If you want, i in can delete this line